drivers/wifi/eswifi/eswifi_socket_offload.c

```diff
@@ -106,8 +106,8 @@ static int eswifi_socket_listen(void *obj, int backlog)
 	return ret;
 }
 
-void __eswifi_socket_accept_cb(struct net_context *context, struct sockaddr *addr,
-			       size_t len, int val, void *data)
+static void __eswifi_socket_accept_cb(struct net_context *context, struct sockaddr *addr,
+				      socklen_t len, int val, void *data)
```

Comment:
`uint32_t` is sufficient to represent any socket length, so rather than mixing up POSIX in drivers, I would suggest sticking to fixed-width integer types. That also mirrors what is done in the network subsystem.

```suggestion
				      uint32_t len, int val, void *data)
```